pytest_tests/constants.py

```diff
@@ -1,3 +1,3 @@
-CLIENT_ID = 15033
-CLIENT_SECRET = "dSbJw0V5RWbl3rBlOksmtNGHFPsz0PJDHlREOPPS"
-REDIRECT_URI = "http://localhost:8080/callback"
+CLIENT_ID =
```

Comment:
should use os.getenv or something to define the credentials

Reply:
solved in 4b612f268394dace26a0a655e4b9711d65d92bbe